DigitalLearningSolutions.Web/Views/Supervisor/Shared/_AssessmentQuestionReviewCells.cshtml

```diff
@@ -1,6 +1,9 @@
 ﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments;
 @model AssessmentQuestion
-
+@{
+  var actionLinkText = Model.Requested != null && Model.Verified == null ? "Confirm" : "View";
+  if (!String.IsNullOrEmpty(Model.SupportingComments)) actionLinkText += " (comments)";

```

Comment:
This also needs to check for Model.SupervisorComments not being null, i.e:
`if (!String.IsNullOrEmpty(Model.SupportingComments)|!String.IsNullOrEmpty(Model.SupervisorComments )) actionLinkText += " (comments)";`

Reply:
Added that change and confirmed `Model.SupervisorComments` in _AssessmentQuestionReviewCells.cshtml is retrieved from SelfAssessmentResultSupervisorVerifications table.